frontend/src/app/features/assistance-selection/assistance-selection.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AssistanceSelectionComponent } from './assistance-selection.component';
+
+describe('AssistanceSelectionComponent', () => {
+  let component: AssistanceSelectionComponent;
+  let fixture: ComponentFixture<AssistanceSelectionComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ AssistanceSelectionComponent ]
+    })
+    .compileComponents();
+
+    fixture = TestBed.createComponent(AssistanceSelectionComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should have no level selected at first', () => {
+    expect(component.selected).toEqual('')
+  })
+
+  it('handleSelectLevel should change the selected level', () => {
+    const key: string = "TEST"
```

Comment:
Semicolons ... all through this file.

Reply:
(they're missing)

Reply:
My habits of relying on a linter are coming back to bite me now... Fixed.